mixins/harvester-vm/index.js

```diff
@@ -608,7 +608,7 @@ export default {
         userDataJson.ssh_authorized_keys = this.getSSHListValue(this.sshKey);
       }
 
-      if (userDataJson.ssh_authorized_keys && userDataJson.ssh_authorized_keys.lenght === 0) {
+      if (userDataJson.ssh_authorized_keys && userDataJson.ssh_authorized_keys.length === 0) {
```

Comment:
??

Reply:
I expect that this would always evaluate `false`..

Reply:
I guess that means that if you have `userDataJson = {ssh_authorized_keys: []}` then it'll stay that way instead of having the property removed.

I suppose there could be code that isn't properly assuming the field might not be present...